package.json

```diff
@@ -86,9 +86,6 @@
     "sass-loader": "^10.1.1",
     "start-server-and-test": "^1.14.0"
   },
-  "resolutions": {
```

Comment:
please remove changes to package.jso and yarn.lock

Reply:
Done :)